adapt/__init__.py

```diff
@@ -1,3 +1,18 @@
+# Copyright 2018 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 __author__ = 'seanfitz'
```

Comment:
I have dropped __author__ lines from mycroft-core.  I think they are confusing in the maintenance world -- do you keep all the names in place forever, do you remove names after a contributors work is removed from the file, etc.  In my opinion Github and the contributor list memorializes this far better than an __author__ line.

@clusterfudge Any heartburn about removing all of the __author__ lines?